src/webnn_native/openvino/GraphIE.cpp

```diff
@@ -801,6 +807,36 @@ namespace webnn_native { namespace ie {
         return {};
     }
 
+    MaybeError Graph::AddSplit(const op::Split* split) {
+        auto input = mGraphNodeMap[split->Inputs()[0].Get()];
+        const ngraph_node_t* axisNode =
+            AddConstantWithGraph<int32_t>(precision_e::I32, {}, {split->GetAxis()});
+
+        ngraph_node_t* outputNodes;
+        std::vector<uint32_t> splits = split->GetSplits();
+        IEStatusCode status = IEStatusCode::OK;
+        if (splits.size() == 1) {
+            status = ngraph_split(input, axisNode, splits[0], &outputNodes);
+        } else {
+            ngraph_node_t* splitsNode =
+                AddConstantWithGraph<uint32_t>(precision_e::U32, {splits.size()}, splits);
+            status = ngraph_variadic_split(input, axisNode, splitsNode, &outputNodes);
+            ngraph_node_free(&splitsNode);
+        }
+        DAWN_TRY(CheckStatusCode(status, "ngraph split"));
+        uint32_t number;
+        status = ngraph_get_output_number(outputNodes, &number);
+        DAWN_TRY(CheckStatusCode(status, "ngraph get output number"));
+        for (uint32_t i = 0; i < number; ++i) {
```

Comment:
Need we check `number == split->output().szie()`?

Reply:
That's good assert, done, thanks.